src/openforms/forms/models/form_variable.py

```diff
@@ -47,7 +49,7 @@ def create_for_form(self, form: "Form") -> None:
         for form_step in form_steps:
             self.create_for_formstep(form_step)
 
-    def create_for_formstep(self, form_step: "FormStep") -> list["FormVariable"]:
+    def create_for_formstep(self, form_step: FormStep) -> list[FormVariable]:
```

Comment:
I'll manage the overlap in a separate commit, I think this can either be refactored or removed now with `synchronize_for`